### DISCUSSION TITLE
DLL problem

### DISCUSSION THREAD

**Discussion by Geant200** [kus/cs2-modded-server#112] [Q&A]
Hello everyone I hope you can help me I don't know where to ask for help.

I have a dedicated server under Windows Server 2022. I encounter a problem launching the server.

It tells me that I have problems with DLL files but when I look to see if the files are present they are all there I don't understand can you help me understand what I forgot?.

![1](https://github.com/kus/cs2-modded-server/assets/15239514/d4b09dac-d2ec-405e-828f-93e7b8a7e2d0)
![2](https://github.com/kus/cs2-modded-server/assets/15239514/8032b4ff-c552-46b4-bdc6-93b87e47cf5d)

thank you in advance.

**Comment by kus** [comment]
Hi @Geant200 it doesn't look like you are using this repo for your server.

Can you try follow [these instructions](https://github.com/kus/cs2-modded-server?tab=readme-ov-file#running-on-windows) and if you run into issues we can try help you.
